Module10/Task1/Task1Library/XmlFileSerializer.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.IO;
+using System.Xml.Serialization;
+
+namespace Task1Library
+{
+    public class XmlFileSerializer
+    {
+        public void SaveToXmlFile(Catalog catalog, string fileName)
+        {
+            using (var fs = new FileStream(fileName, FileMode.Create))
+            {
+                // var formatter = new XmlSerializer(typeof(Catalog));
+                // formatter.Serialize(fs, catalog);
+
+                //XmlSerializerNamespaces ns = new XmlSerializerNamespaces();
+                //ns.Add("", "");
+                //XmlSerializer s = new XmlSerializer(typeof(Catalog));
+                //s.Serialize(fs, catalog, ns);
+
+                var ns = new XmlSerializerNamespaces();
```

Comment:
Bad name for variable

Reply:
My mistake. Long understood how to make the necessary namespaces, tried different options. And I forgot about the code refactoring in this class, and remained the commented code and naming problems